src/main/webapp/content/scss/themes/theme-default.scss

```
@@ -39,6 +39,11 @@ html {
     color: $link-color;
 }
 
+.btn-check:checked + .btn,
+btn:hover {
+    color: white !important;
+}
+
```

IMO using higher CSS specificity is preferrable over using `!important`. Also, I think the second rule has no effect because there is no `<btn>` HTML element.

```suggestion
.btn-check:checked+label.btn {
    color: white !important;
}

```

---

I think @pal03377 is right here. Additionally, please document in a comment what you're doing here and what you are selecting.